ahfe_tutorial/python_tutorial.ipynb

```diff
@@ -0,0 +1,930 @@
+{
```

Comment:
<p>Then above this cells we could say:</p><p> </p><p>"You can save the <code>AbsoluteSolvationProtocolResult</code> to a JSON output file in the following manner:" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>